frontend/src/HomePage/HomePage.jsx

```diff
@@ -171,13 +171,16 @@ class HomePageComponent extends React.Component {
     this.setState({ showAppDeletionConfirmation: true, appToBeDeleted: app });
   };
 
-  cloneApp = async (appId, appName) => {
+  cloneApp = async (appName, appId) => {
```

Comment:
AppCard.jsx line 139, this function is called with 
`cloneApp(app)` app object as only attribute

Reply:
Removed that cloneApp func call, it was not needed.